src/main/java/types/Display.java

```diff
@@ -196,5 +196,5 @@ public interface Display {
      * @status added
      * @since 4.5
      */
-    Integer disconnectActionDelay();
+    Short disconnectActionDelay();
```

Comment:
the question is how strict we want to be here in terms of backward compatibility
as we've reached the GA of oVirt, we shouldn't make changes to the API that break backward compatibility. you can argue that values higher than short were handled incorrectly, but they were still accepted.
my point is that it would have been best to model it as short everywhere from the beginning, but given the current situation we probably better keep it as integer here and change the database layer to support that

Reply:
We should maintain backward compatibility to prevent breaking a behavior that is already used. Do you think that overflow of values out of `Short` range is really used? This is very unlikely. It is very unlikely that very large values are used at all, but OK. But the overflow behavior, I think, we can change without risk of breaking anything.

Reply:
I don't think that there's any real-world scenario where a number over 9 hours would be used. So on the one hand, we're not really risking breaking backwards-compatibility, on the other hand we're not solving any *real* problem either. I think I prefer not to merge this, if not for any other reason, because it would be the first and only place in the API where Short is used, and if there's no good reason why introduce something that deviates from the norm